packages/ui/src/i18n/dictionaries/authenticator/nl.ts

```diff
@@ -3,6 +3,8 @@ import { AuthenticatorDictionary } from './types';
 export const nlDict: AuthenticatorDictionary = {
   'Account recovery requires verified contact information':
     'Accountherstel vereist geverifieerde contactgegevens',
+  'Attributes did not conform to the schema: email: The attribute is required\n':
```

Comment:
Unfortunately we do not have control over this message and cannot include this key in our translations